.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "build_library/secure_demo/shim"]
```

Comment:
Please use the repo manifest instead. It is going to be confusing to use both git submodules and repo.